operator/src/main/java/oracle/kubernetes/operator/DomainStatusUpdater.java

```diff
@@ -1326,6 +1326,27 @@ void modifyStatus(DomainStatus status) {
     }
   }
 
+  public static class UpdateCompletedConditionSteps extends StatusUpdateStep {
```

Comment:
If this always sets completed/false, a name that indicates that would be clearer than one which just notes that the step _updates_ the condition.

Reply:
what about ClearCompletedConditionStep?